queries/wiki.py

```diff
@@ -139,9 +148,18 @@ def QWikiQuery(node, params, result):
     result.qkey = result["subject_nom"]
 
 
+def QWikiSubjectNf(node, params, result):
```

Comment:
A more concise way to do this is to name the nonterminal `QWikiSubject/fall (QWikiSubject_nf, _þf, _þgf)`; then you only need one `def QWikiSubject(node, params, result)` function for all variants.